src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0, 
+        maximum-scale=1.0, 
+        minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;1,500&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img 
+          class="card__image-iMac"
+          src="images/imac.jpeg"
+          alt="computer"
+        >
+      </div>
+      <main class="main">
+        <h2 class="main__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
```

Comment:
```suggestion
        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
```
BEM? у тебя должно быть 2 BEM блока `card` и `stars `